qiskit/primitives/primitive_job.py

```diff
@@ -46,19 +48,30 @@ def _submit(self):
         self._future = executor.submit(self._function, *self._args, **self._kwargs)
         executor.shutdown(wait=False)
 
+    def _prepare_dump(self):
```

Comment:
I recall that this method is used when we serialize a job. But, this method is not used any other methods of this class. So, it might be good to add a comment just in case though it is a private method.

Reply:
Added in https://github.com/Qiskit/qiskit/pull/12963/commits/fac959a4f7deaf6375d23f8293c92fe2119ecdd6